<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Quanli</class>
 <widget class="QDialog" name="Quanli">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/nenchu1/picture/troitoi.jpg);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 32pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Manage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSave">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/form/picture/nenxanhgiaodich.png);
border-image: url(:/form/picture/nenxanhgiaodich.png);
color: rgb(255, 255, 255);
font: 600 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Save Account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border-image: url(:/form/picture/cam.png);
background-image: url(:/form/picture/cam.png);
font: 600 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Add Account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/form/picture/nenxanhgiaodich.png);
border-image: url(:/form/picture/nenxanhgiaodich.png);
color: rgb(62, 62, 62);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::MailReplyAll"/>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetFiles">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>211</y>
      <width>211</width>
      <height>269</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditContent">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>211</y>
      <width>681</width>
      <height>269</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditNewUser">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>681</width>
      <height>269</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonHash">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/form/picture/nenxanhgiaodich.png);
border-image: url(:/form/picture/nenxanhgiaodich.png);
color: rgb(255, 255, 255);
font: 600 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Hash Password</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditHash">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>681</width>
      <height>269</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/form/picture/nenxanhgiaodich.png);
border-image: url(:/form/picture/nenxanhgiaodich.png);
color: rgb(62, 62, 62);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::SystemLogOut"/>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
